palindrom

```diff
@@ -0,0 +1,34 @@
+function pal(start , end , thisword){
+
+   
+    if( thisword[start] != thisword[end] ){
+        console.log(thisword[start]);
+        console.log(thisword[end]);
+        return false;
+    }
+   
+    if(start != end){
+        pal(start+1 , end-1 , thisword);
+        console.log("plaindrom");
+    }
+    
+    return true;
+}
+
+function isPalindromeFunction(str)
+{
+    let l = str.length
+     
+    // An empty string is
+    // considered as palindrome
+    if (l == 0 || l == 1)
+        return true;
+     
+    return pal( 0 , l - 1 , str);
+}
+
+let str="deified";
```

Comment:
As mentioned before, we shouldn't include values we tested with in the implementation file, only the implementation. If you would like to include tests, make a new file containing the tests only.